src/main/java/bio/terra/service/filedata/FSFileInfo.java

```diff
@@ -33,12 +33,12 @@ public FSFileInfo createdDate(String createdDate) {
     return this;
   }
 
-  public String getGspath() {
-    return gspath;
+  public String getCloudPath() {
+    return cloudPath;
```

Comment:
this is cool but I just want to make sure that this continues to work with existing data in firestore.  I think we left that model with gsPath but I just want to make sure we validate when we merge and deploy on dev